src/Protocol/Protocol_1_9.cpp

```diff
@@ -1724,9 +1724,8 @@ bool cProtocol_1_9_0::CompressPacket(const AString & a_Packet, AString & a_Compr
 
 int cProtocol_1_9_0::GetParticleID(const AString & a_ParticleName)
 {
-	static bool IsInitialized = false;
 	static std::map<AString, int> ParticleMap;
-	if (!IsInitialized)
```

Comment:
Is this `IsInitialized` variable set anywhere else? Did you remove all its occurrences?